src/history.cpp

```diff
@@ -11,17 +12,17 @@ namespace mamba
     std::vector<History::ParseResult> History::parse()
     {
         std::vector<ParseResult> res;
-        fs::path history_file_path = fs::path(m_prefix_data) / "conda-meta" / "history";
-        LOG_INFO << "parsing history: " << history_file_path;
+        m_history_file_path = fs::path(m_prefix_data) / "conda-meta" / "history";
+        LOG_INFO << "parsing history: " << m_history_file_path;
 
-        if (!fs::exists(history_file_path))
+        if (!fs::exists(m_history_file_path))
```

Comment:
Decided to create a member in the class because I'd have to do the same line:
`         fs::path history_file_path = fs::path(m_prefix_data) / "conda-meta" / "history";` again in the method I added.